### PR TITLE
Implementing Library-app logic

### PR DESCRIPTION
**Pull request by CalyCherkaoui**
In this PR, we implemented the Library-app logic.
The features implemented are:
- Creating a book by entring its data
- Deleting a book from the library(book list)
- Toggling the 'read' status of the book via a button in the display
- Using LocalStorage to store data in the local machine

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        </button>
+      </div>
+      <!-- <div id="form_submission" class="hide_element">
+        <button id='submit_book_data'>
+          Submit Book info
+        </button>
+        <button id='cancel_book_data'>
+          Cancel
+        </button>
+      </div> -->
```
Comment (uche-inyama) [inline index.html]: Remove commented lines of code.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        <th>read</th>
+        <th>delete</th>
+      </tr>
+
+    </table>
+
+    <div class="user_input">
+      <button id="addbook_btn">
+        <!-- <span id="addbook_txt">ADD NEW BOOK</span>
+        <span id="cancel_txt">CACEL</span> -->
```
Comment (uche-inyama) [inline index.html]: Remove commented lines of code.

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Changes Required

### Hi team :wave: 

Nice work, just few updates to make.

- [x] Keep the application logic separated from DOM manipulation tasks ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/dom_logic_separation_requirement.png))

- [x] I have left some comments below about removing commented lines of code.

That's it, make these changes, then submit another form for a code review.

### ``` Have a great day ``` :slightly_smiling_face: 

``` Uchechukwu Henry Inyama ```

File: index.html (hunk 1 above)
Comment (CalyCherkaoui) [reply index.html]: Solved! Thanx

File: index.html (hunk 2 above)
Comment (CalyCherkaoui) [reply index.html]: Solved! Thanx

**Review by uche-inyama — APPROVED** [review]
## Approved

### Hi team :wave: 

Go ahead with your next task.

### ``` Have a great day ``` :slightly_smiling_face: 

``` Uchechukwu Henry Inyama ```

**Pull request merged**